src/Plugin/Validation/Constraint/EmbeddableConstraint.php

```diff
@@ -15,6 +15,6 @@
  */
 class EmbeddableConstraint extends OEmbedResourceConstraint {
 
-  public $oEmbedNotAllowed = 'An oEmbed link is not permitted on an Embeddable which has a custom Embed Code.';
+  public $embedCodeNotAllowed = 'The given embeddable code is not permitted.';
```

Comment:
I'd vote to link to a help article here for clarity, but that's just a nice-to-have.